assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,35 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
```

Comment:
Good attempt at the stretch! There are a couple of issues with your syntax and your returns. We can review this later.